assemble/conf/accumulo-env.sh

```diff
@@ -94,6 +94,7 @@ case "$cmd" in
   tserver) JAVA_OPTS=("${JAVA_OPTS[@]}" '-Xmx768m' '-Xms768m') ;;
   compaction-coordinator) JAVA_OPTS=("${JAVA_OPTS[@]}" '-Xmx512m' '-Xms512m') ;;
   compactor) JAVA_OPTS=("${JAVA_OPTS[@]}" '-Xmx256m' '-Xms256m') ;;
+  sserver) JAVA_OPTS=("${JAVA_OPTS[@]}" '-Xmx512m' '-Xms512m') ;;
```

Comment:
When testing running scan servers in Kubernetes using this [configuration](https://github.com/keith-turner/accumulo-testing/blob/322b32c62113b974b9dd18404d0e57902dc13b8e/sstest/accumulo-scanservers.yaml#L92) I had to make the following change inorder to be able to set the memory in the deployment descriptor.

```suggestion
  sserver) JAVA_OPTS=( '-Xmx512m' '-Xms512m' "${JAVA_OPTS[@]}") ;;
```

I don't think we should apply this change in this branch. Thinking if the change makes sense that it should be applied to all the server config as its own PR.  Just wanted to comment here in case anyone is running Kubernetes tests.